.eslintrc.js

```diff
@@ -103,6 +103,10 @@ module.exports = {
         "@typescript-eslint/no-unsafe-assignment": "off",
         // TODO: reEnable no-unsafe-member-access
         "@typescript-eslint/no-unsafe-member-access": "off",
+        "@typescript-eslint/restrict-template-expressions": "off",
+        "@typescript-eslint/no-unused-vars": "off",
+        // TODO: reEnable explicit-module-boundary-types
```

Comment:
Reason to disable this rule?

Reply:
For no-unused-vars, If we turn on this rule, we need to remove all unused vars in function signature. I remembered once you mentioned there are some params in function signature which is not currently used in logic, but will be used in future, thus you leave them in function signature for now. Also, it's related to anther comment you gave below, while implementing an abstract class's method, this lint rule will report error if param declared but not used. 

For restrict-template-expressions, I think we need to bring it back for sure. There are some lint error in current code base related to this rule, the fix would need some context such as: https://github.com/alexa/ask-toolkit-for-vscode/blob/master/src/utils/urlHandlers.ts#L57. I was thinking to disable it for now, thus developer contributor won't be bothered by current lint issues and at the same time we could continually fix all lint rules. Probably add a TODO as well ?

Reply:
A TODO would be a good start.

Reply:
Got it, will add a TODO